labman/db/process.py

```diff
@@ -2013,6 +1995,9 @@ def compute_pooling_values_minvol(
         sample_vols: np.array of floats
             the volumes in nL per each sample pooled
         """
+
+        old_err = np.seterr(divide='ignore')
```

Comment:
I would suggest using the context manager version of this, as it makes it clear what block of code this directive is for:

```python
with np.errstate(divide='ignore'):
    # divisions and more!
```